legend-engine-pure-code-compiled-core/src/main/resources/core/pure/runtime/runtimeExtension.pure

```diff
@@ -23,7 +23,7 @@ Class meta::pure::runtime::EngineRuntime extends Runtime
 // NOTE: we don't need to process the rest of engine runtime for now
 }
 
-Class meta::pure::runtime::PackageableRuntime extends PackageableElement
+Class meta::pure::runtime::PackageableRuntime extends Runtime, PackageableElement
 {
    runtimeValue: meta::pure::runtime::EngineRuntime[1];
```

Comment:
lets check if we really need this or we can get rid of it entirely, by name PackageableRuntime should just extend PackageableElement and Runtime.
If its too much effort we can do that in separate PR but lets check once

Reply:
I think we shouldn't be extending it to Runtime, because we add in a runtimeValue